packages/backend/src/execution/execute.test.ts

```diff
@@ -331,6 +337,58 @@ const languageProvider: LanguageQueryProvider = {
     },
 };
 
+const paymentProvider: PaymentQueryProvider = {
```

Comment:
@jimezesinachi is there a reason you added all these query providers?

Reply:
I don't mind having them, I'm just wondering if I'm missing a connection between this and defer() queries